src/helpers/ens.ts

```diff
@@ -0,0 +1,20 @@
+import { Contract } from 'ethers';
+import { web3Provider } from '@rainbow-me/handlers/web3';
+import { ENSABI, ETHRegistrarControllerABI } from '@rainbow-me/references';
```

Comment:
should we use https://github.com/ensdomains/ens-contracts rather?

Reply:
yeah that would be a way to go but since I'm just getting the ABIs from there it didn't make sense to include the whole package into the app, WDYT?

Reply:
Yeah true, you can also import each contract separately:

```
import ENS from '@ensdomains/ens-contracts/build/contracts/ENS.json'
```